backend/plugins/frontline_api/src/modules/reports/graphql/reportQueries.ts

```
@@ -289,12 +301,23 @@ export const reportQueries = {
     });
 
     pipeline.push(
+      {
+        $addFields: {
+          createdAtDate: {
+            $cond: {
+              if: { $eq: [{ $type: '$createdAt' }, 'string'] },
+              then: { $dateFromString: { dateString: '$createdAt' } },
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)